docs/kanban.md

```diff
@@ -0,0 +1,164 @@
+
+#### `priority-backlog`
+
+This is an ordered list of categorized cards that the team is considering for the next work. The priority is based on impact, user value, urgency and the current team plans and capacity. The team is revisiting the epic-level priorities on a quarterly level and the cards in the `priority-backlog` should be done in 3 months. This means the number of cards needs to be maintained below ~50.
+
+#### `refined`
+
+This column consists of cards prepared to be worked on next when someone has a free capacity. The tasks should be actionable right away and doable by anyone. To get a card to this state a [refining process](#Refine) is used.
+
+#### `in-progress`
+
+When a card is taken by someone from a `refined` column, it is assigned to that person and moved to this column. This is the list of cards that are being actively worked on.
+
+#### `in-review`
+
+These tasks are nearly finished, being reviewed and polished.
+
+#### `done`
+
+This is the column where all the done cards result.
+
+### Card labels
+
+Here is a list of labels we use to categorize cards to help ourselves navigate through the backlog and plan our work.
+Note that there is no priority label since this consists of multiple factors like _impact_ and _gain_. Combined with the urgency, our current plans (based on demand) and capacity, this is visible from the place on the board.
+
+#### Area
+
+These labels help to group related cards across all the projects. The area can determine a target git-forge (e.g. `area/github`/`area/gitlab`), service being integrated (e.g. `area/testing-farm` or `area/copr`) or a shared code-level logic (e.g. `area/config` or `area/database`).
+
+#### Complexity
+
+This is a rough estimation of how much time is expected for this card to be done. Mainly to separate epics from single cards. -- work consisting of multiple tasks, usually taking more time and being discussed in quarterly meetings.
+
+#### Gain
+
+This determines the value it gives to affected users.
+
+- `gain/high`: significant change for the user, e.g.:
+  - New users can start using Packit.
+  - Significant time is saved for the user when the task is done.
+  - A user might stop using Packit when not available.
+- `gain/low`: change not so important to start/stop using Packit, e.g.:
+  - If `workaround-exists` (separate label).
+  - User can fully benefit from Packit without the card being done.
+
+#### Impact
+
+These labels determine the size of a user group affected by this card combined with a frequency. To make this comparable, it is based on generic user roles like `upstream-developer` or `fedora-maintainer`. This means we won't mark a card as high-impactful when all (but a few in reality) users of a rear functionality are affected.
```

Comment:
I assume this is supposed to be rare? Rear functionality had me very confused for a while :sweat_smile: 

```suggestion
These labels determine the size of a user group affected by this card combined with a frequency. To make this comparable, it is based on generic user roles like `upstream-developer` or `fedora-maintainer`. This means we won't mark a card as high-impactful when all (but a few in reality) users of a rare functionality are affected.
```